### PR TITLE
Add API test for Organization → User association - issue #1406

### PR DESCRIPTION
**Pull request by abalakh**
Added API test to check if it's possible to associate user
with organization

### CODE REVIEW

File: tests/foreman/api/test_organization.py [hunk 1]
```diff
@@ -246,6 +246,26 @@ def test_positive_update(self, attrs):
...
+
+        client.put(
+            self.organization.path(),
+            {'organization': {'user_ids': [user_id]}},
+            verify=False,
+            auth=get_server_credentials(),
+        ).raise_for_status()
+
+        new_attrs = self.organization.read_json()
+        self.assertEqual(user_id, new_attrs['users'][0]['id'])
```
Comment (Ichimonji10) [inline tests/foreman/api/test_organization.py]: It's also good to check that only one user is associated with `self.organization`. You can make an assertion like this:

```python
attrs = self.organization.read_json()
self.assertEqual(1, len(attrs['users']))
self.assertEqual(user_id, attrs['users'][0]['id'])
```

File: tests/foreman/api/test_organization.py [hunk 2]
```diff
@@ -246,6 +246,26 @@ def test_positive_update(self, attrs):
...
+        """
+        user_id = entities.User().create()['id']
+
+        client.put(
+            self.organization.path(),
+            {'organization': {'user_ids': [user_id]}},
+            verify=False,
+            auth=get_server_credentials(),
+        ).raise_for_status()
+
```
Comment (Ichimonji10) [inline tests/foreman/api/test_organization.py]: Nitpick: drop the whitespace in this method. Whitespace is useful when it denotes significantly distinct hunks of code. This method is very small, and the whitespace needlessly makes the reader slow down and wonder what is being communicated.

**Comment by Ichimonji10** [comment]
ACK pending comments.

By the way, can you post output from a test run with this method? It's nice to see proof that the test method runs successfully. You can just copy-and-paste the output of a command like `nosetests tests/foreman/api/test_organization.py -m test_associate_user_with_organization`.

File: tests/foreman/api/test_organization.py (hunk 1 above)
Comment (elyezer) [inline tests/foreman/api/test_organization.py]: :+1:

**Comment by elyezer** [comment]
ACK, pending @Ichimonji10's comments.

**Comment by abalakh** [comment]
Updated, thanks for comments. Here's test run output:
```
$ nosetests tests/foreman/api/test_organization.py -m test_associate_user_with_organization
.
----------------------------------------------------------------------
Ran 1 test in 10.177s

OK
```

**Pull request merged**